hw/ip/kmac/data/kmac_base_testplan.hjson

```diff
@@ -2,7 +2,7 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 {
-  name: kmac_base
```

Comment:
Alternately, you could get rid of the `name` field in the base testplan and revert all other changes.

Reply:
That's true, but I think then each manual test wouldn't be named with the `variant` as well - wdyt?

Reply:
Lets file an issue to sort this out later. The automated and manual tests are prefixed differently, which is not ideal.

Reply:
agreed!